src/styles/foundation/_focus-ring.scss

```diff
@@ -5,10 +5,10 @@
 ///
 
 @mixin focus-ring($size: 'base', $offset: rem(1px), $style: 'base') {
+  $stroke: rem(2px);
+  $position: calc(#{$offset} + #{$stroke});
+
   @if $style == 'base' {
-    $easing: cubic-bezier(0.4, 0.22, 0.28, 1);
```

Comment:
Was `$easing` not used?

Reply:
I'm not sure, I just noticed it wasn't used. I'm following this PR up with some focus ring work so I'll look into it then